### DISCUSSION TITLE
Сервер svodd.ru работает с ошибками. Проблемы с резолвом доменных имен

### DISCUSSION THREAD

**Discussion by audetv** [audetv/cohesion#14] [Announcements]
В работе сервисов облачного провайдера Selectel произошел сбой.

На данный момент не происходит обновления комментариев в теме
https://svodd.ru/bolshaya-svoddnaya-tema?page=1&sort=-date

https://searchengines.guru/ru/forum/1092238

[Мониторинг](https://selectel.live/incidents/181896#activity-id-366046)
Причиной проблемы являются глобальные неполадки в ru-зоне доменных имен, что может затрагивать работу всех сервисов, обращение к которым осуществляется с помощь доменов в зоне .RU. В свою очередь, эта проблема затронула не только часть сервисов Selectel, но другие сервисы различных провайдеров.

Мы постараемся держать вас в курсе ситуации.

January 30, 2024 · 19:21
[Проблема](https://selectel.live/incidents/181896#activity-id-366032)
На данный момент фиксируются глобальный инцидент в работе DNS в зоне .ru.

Временным решением является подключение к сервисам по IP адресу.

Мы уже работаем над устранением проблемы.
Приносим извинения за доставленные неудобства.

January 30, 2024 · 18:48

**Comment by iprst** [comment]
Видимо, имена уже разрешились, поскольку и на сайте и в телеге вроде бы все комментарии отобразились. Или это зависит ещё и от провайдера клиента? В общем, у меня работает.

**Comment by audetv** [comment]
Да все заработало. Пока понаблюдаю. Да вроде все и в телегу прилетело и на сайт 
Я думаю теперь должно работать, т.к. ранее на мобильном сайт не открывался, не было телеги и соединения нод серверов с сайтами и хостинг панели висели. не могли получит данные по api. Сейчас вроде ок.

**Reply by audetv** [reply]
У меня не было много времени разбираться, узнал о сбое в пути, пока приехал пока посмотрел проверил, понял, что по IP сервера друг друга видят, но когда происходило обращения через доменные имена, а это все сервисы api, то были сбои, Загуглил увидел, что есть проблема в сети, написал сообщение. 

В начале я минут 30 не мог понять, что упало. Вроде пингуется, но не работает. Сервера недоступны, друг друга не видели. И видимо зависело от того где и как ДНС  записи локальные (по России) разрешались. Не знаю, судя по комментариям разные хостинги пострадали. Т.к. наш сайт не сам в себе, а с разных мест данные берет, то мы и получили сбой в обновлении.

**Reply by iprst** [reply]
Я в теме тоже написал возможную причину из наблюдений за несколькими днями, а здесь не буду по соображениям здоровья ))

**Comment by audetv** [comment]
[Мониторинг](https://selectel.live/incidents/181896#activity-id-366091)
На текущий момент сервисы должны работать в штатном режиме.

Мы продолжаем следить за ситуацией.

January 30, 2024 · 21:09

**Comment by audetv** [comment]
[Решено](https://selectel.live/incidents/181896#activity-id-366101)
Проблема решена. Услуги работают в штатном режиме.

January 30, 2024 · 22:24
